### DISCUSSION TITLE
New beginner? I'm asking for directions?!

### DISCUSSION THREAD

**Discussion by Tjay320** [community/community#35662] [New to GitHub]
I'm simply asking for some help?

**Comment by official-mywebprojects** [comment]
Hello @Tjay320, can you let us know what exactly you need help with? This will help people give you specific answers.

However, for all you need to know about getting started with github as a beginner, kindly refer to this guide. https://docs.github.com/en

**Reply by Tjay320** [reply]
What I was asking was a little help learning about coding and how to process it onto algorithms and paste...

**Comment by jfullstackdev** [comment]
if you're asking about GitHub, you may want to check this out, my answer from another thread 

https://github.com/community/community/discussions/31642#discussioncomment-3536733

**Comment by rroossyyiidd** [comment]
sure, you can learn about coding from documentation code

**Reply by 2209199** [reply]
tell me how to hack instagram in this program

**Reply by 2209199** [reply]
can you help

**Reply by Tjay320** [reply]
> sure, you can learn about coding from documentation code

Well I need help hacking into FB and I'm on the GitHub mobile app... Please help... Thank you

**Comment by LucasHT22** [comment]
look this: https://skills.github.com/
